release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-chrony-chrony-default"]
+==== Change to custom `chrony.conf` default on cloud platforms
+
+If a cloud administrator has already set a custom `/etc/chrony.conf` configuration, {op-system} no longer sets the `PEERNTP=no` option by default on cloud platforms. Otherwise, the `PEERNTP=no` option is still set by default. 
```

Comment:
Maybe worth linking to the BZ that the upstream PR addressed - https://bugzilla.redhat.com/show_bug.cgi?id=1924869

Reply:
Added BZ, thanks.